js/form.js

```diff
@@ -0,0 +1,42 @@
+window.addEventListener('DOMContentLoaded', function() {
+    'use strict';
+
+    //Get the form elements defined in your form HTML above
+    const form = document.querySelector('form');
+    const button = document.querySelector('input[type="submit"]');
+    const status = document.getElementById('form-submit-status');
+
+    //Success and Error functions for after the form is submitted
+    function success() {
+        form.reset();
+        status.textContent = "Thanks! We'll be getting back to you in 24-48 hours.";
+        button.style.display = 'none';
+    }
+
+    function error() {
+        status.textContent = 'Oops! There was a problem.';
+    }
+
+    // Handle the form submission event
+    form.addEventListener('submit', function(event) {
+        event.preventDefault();
+        const data = new FormData(form);
+        ajax(form.method, form.action, data, success, error);
```

Comment:
We could combine these two lines and do `form.method, form.action, new FormData(form)` as our arguments.

I think we could also remove `success` and `error` from our arguments and parameters. The functions are already visible to the scope of the `ajax` function, so passing them in is redundant.